### PR TITLE
Fix unit-test

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by chtyim** [comment]
Ran the test locally

**Comment by sagarkapare** [comment]
LGTM

**Comment by chtyim** [comment]
Building in http://builds.cask.co/browse/CDAP-DUT4626-1

File: cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java [hunk 1]
```diff
@@ -93,7 +95,7 @@ public Injector getInjector() {
 
     return Guice.createInjector(
       new ConfigModule(cConf, hConf),
-      new AbstractModule() {
+      Modules.override(new LocationRuntimeModule().getInMemoryModules()).with(new AbstractModule() {
```
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java]: why use inMemoryModules for DataMigration?

Comment (chtyim) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java]: It was like that before. Gotta ask @rohitsinha54

Comment (chtyim) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java]: Hum... looking it closer, seems like the old binding was just wrong.

Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/DataMigration.java]: I think @sagarkapare worked on DataMigration.

**Comment by chtyim** [comment]
Unit-test running in http://builds.cask.co/browse/CDAP-DUT4626-3

**Comment by anwar6953** [comment]
lgtm

**Comment by chtyim** [comment]
Build passed.

**Pull request merged**